src/backend/marsha/bbb/tests/api/classroomdocument/test_upload_ended.py

```diff
@@ -40,7 +40,7 @@ def assert_user_cannot_end_an_upload(self, classroom, classroom_document, token)
             f"{classroom_document.pk}/upload-ended/",
             {
                 "file_key": f"classroom/{classroom.pk}/classroomdocument/"
-                f"{classroom_document.pk}/4564565456",
+                f"{classroom_document.pk}/foo.pdf",
```

Comment:
I believe this patch should belong to the previous commit :)

Reply:
Oh good catch! Changed it :sweat_smile: